src/package/packagePushUpgrade.ts

```diff
@@ -185,13 +201,14 @@ export class PackagePushUpgrade {
         .map(
           (job, index) =>
             // eslint-disable-next-line @typescript-eslint/no-unsafe-member-access
-            `Job ${index + 1} Error:${JSON.stringify(job?.sf__Error, null, 2)}`
+            `Job ${index + 1}: Error:${JSON.stringify(job?.sf__Error, null, 2)}`
```

Comment:
If `job.sf__Error` is ever undefined this will throw an error.  Might be better to not disable the lint rule and ensure it exists in code unless you are 100% sure it will always be defined at this point.